test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/MutableObjectModelBinderTest.cs

```diff
@@ -905,14 +905,13 @@ public void ProcessDto_BindRequiredFieldNull_RaisesModelError()
             Assert.Equal(ModelValidationState.Invalid, modelState.ValidationState);
 
             var modelError = Assert.Single(modelState.Errors);
-            Assert.Null(modelError.Exception);
-            Assert.NotNull(modelError.ErrorMessage);
-            Assert.Equal("A value is required.", modelError.ErrorMessage);
+            Assert.Equal(string.Empty, modelError.ErrorMessage);
+            Assert.IsType<NullReferenceException>(modelError.Exception);
         }
 
         [Fact]
         [ReplaceCulture]
-        public void ProcessDto_RequiredFieldMissing_RaisesModelError()
+        public void ProcessDto_MissingDataForRequiredFields_NoErrors()
```

Comment:
A number of these tests have been simplified because we expect the validation system to produce these errors now.